runtime/lua/vim/lsp/rpc.lua

```diff
@@ -624,9 +625,9 @@ end
 --- server process. May contain:
 --- - {cwd} (string) Working directory for the LSP server process
 --- - {env} (table) Additional environment variables for LSP server process
----@returns Client RPC object.
+---@return table|nil Client RPC object.
 ---
----@returns Methods:
+---Client Object Methods:
```

Comment:
the blank line seems to be messing up doxygen's parsing. try starting this line at the end of the @return line.